### ISSUE TITLE
Bash variable substitution issue

### ISSUE THREAD

**Issue by pkittenis** [pkittenis/parallel-ssh#34]
Reproduction:

```python
import pssh
client = pssh.ParallelSSHClient(hosts=['localhost'])
command = """for i in 1 2 3; do echo $i; done"""
list(client.run_command(command)['localhost']['stdout'])
['', '', '']
```

Output should be `[1,2,3]`.

**Comment by pkittenis** [comment]
This is resolved in current master branch.

Pending #24 in order to be able to make a test for this behaviour which is bash-specific.

**Comment by pkittenis** [comment]
Resolved by `0.70.4`
